token/services/ttxdb/README.md

```diff
@@ -0,0 +1,96 @@
+# Token Transactions DB
+
+The Token Transactions DB is a database of audit records. It is used to track the
```

Comment:
shall we reserve the word audit for auditors, and use a different word for owners?